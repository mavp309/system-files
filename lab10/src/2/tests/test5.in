u 12
o -1
o -1
o -1
u 14
u 12
o -1
q 0
