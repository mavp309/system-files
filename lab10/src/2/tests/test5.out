12
Stack empty
Stack empty
12
